cloud-inquisitor/notification/test_text/hijack_test_with_chain.txt

```diff
@@ -0,0 +1,24 @@
+Potential Domain Hijack Found
+
+Deleting resource S3 Bucket:test.bucket.com in AWS account 12345678 has resulted in a potential domain hijack;
+ there is a DNS record pointing to a resource not owned or no longer owned by us.
+
+The following table includes all resources in the chain that would lead to a potential domain hijack.
+
+
+
```

Comment:
Sorry had a thought again, resource 0 and 1 probably make it easy, but should we consider

DNS resource

Target resource
?

Reply:
We could, the generic case I was considering was something like Route53 -> CloudFront -> S3 in which we would need 3 lines. We can definitely make this more sophisticated.

Reply:
ok I think this is fine for now until we understand all the possible flows.